src/main/webapp/app/account/password/password.route.ts

```diff
@@ -8,7 +8,7 @@ export const passwordRoute: Route = {
   path: 'password',
   component: PasswordComponent,
   data: {
-    authorities: [Authority.USER],
+    authorities: [Authority.STUDENT, Authority.TEACHER],
```

Comment:
3 szerep van, nem csak 2

Reply:
de maradhat igy, ha most csak 2 szerepet tamogatunk

Reply:
A 3. szerepet is hozáadatam